### PR TITLE
PICARD-340:  Cover art embedding will overwrite existing ones

### PR DESCRIPTION
**Pull request by rahul-raturi**

### CODE REVIEW

File: picard/tagger.py [hunk 1]
```diff
@@ -451,7 +451,17 @@ def get_files_from_objects(self, objects, save=False):
     def save(self, objects):
         """Save the specified objects."""
         files = self.get_files_from_objects(objects, save=True)
-        for file in files:
+        coverOverwritePrompt = config.setting["cover_overwrite_prompt"]
+        for file, obj in zip(files, objects):
```
Comment (mineo) [inline picard/tagger.py]: I'm not 100% sure about this, but the lengths of `objects` and `files` are not always the same (a track can be associated with multiple files and an album/cluster with multiple tracks).

**Comment by rahul-raturi** [comment]
Hello @mineo,

> I don't embed images and always use the ones from MB, so I'm not sure how useful this is for people without being able to compare the images when the message box is shown.

I don't suppose most people would avoid overwriting the cover art when they are looking to update rest of the album's metadata (that's my personal opinion). I think only a few people who have an embedded cover art from a different source would like such functionality. Still, Picard should have such feature IMHO. As for comparing images when prompt is shown, I couldn't find a way to do so with QMessageBox. Another thing that I came across is that a user can't view the original image after the lookup. So, what I am thinking of doing is displaying the original cover with the latest one when a user right clicks on the track after the lookup and selects `Info...`. That way, a user can compare the covers. I can put a note in the message dialog on how to do so (i.e. right click, and select info ... something like that). Is this approach good enough, or should I try something else?

**Comment by rahul-raturi** [comment]
Umm, @zas, can you remove the "Ready to merge" label please.  I was hoping to get this [visual feedback](https://github.com/metabrainz/picard/pull/474) PR merged before I get back on this. As @mineo said, it would be better to allow comparing images before displaying the prompt.
Besides, I haven't tested it completely, so it requires a little bit more work.

**Comment by samj1912** [comment]
It's been almost 2 years since this was updated. Closing this with a 'Needs more work' tag. You can reopen this if you want to continue working on it. Also we have updated to Qt5 and py3 so be mindful of that while reworking on this PR.

**Pull request closed without merging**